src/common/getters.ts

```diff
@@ -22,22 +22,28 @@ import { AccessGrantGConsent } from "../gConsent/type/AccessGrant";
 import { AccessRequestGConsent } from "../gConsent/type/AccessRequest";
 import { isAccessGrant as isGConsentAccessGrant } from "../gConsent/guard/isAccessGrant";
 import { AccessModes, resourceAccessToAccessMode } from "../type/AccessModes";
+import { AccessGrantOdrl, isCredentialAccessGrantOdrl } from "../odrl";
 
 /**
  * Get the resources to which an Access Grant/Request applies.
  *
  * @example
  *
  * ```
- * const resources = getResources(gConsentGrant);
+ * const resources = getResources(accessGrant);
  * ```
  *
  * @param vc The Access Grant/Request
- * @returns The VC IRI
+ * @returns The resources IRIs
  */
 export function getResources(
-  vc: AccessGrantGConsent | AccessRequestGConsent
+  vc: AccessGrantGConsent | AccessRequestGConsent | AccessGrantOdrl
 ): string[] {
+  if (isCredentialAccessGrantOdrl(vc)) {
+    return vc.credentialSubject.permission.map(
```

Comment:
Can any of these be nullable? Can we get just a single value instead of an array here?

Reply:
According to the type definition. here: https://github.com/inrupt/solid-client-access-grants-js/blob/7f2d36a31943a1af238a20626acfeab925b64bf6/src/odrl/type/AccessGrant.ts#L52, `permission` should be an array and required.